doc/roadmap.rst

```diff
@@ -181,9 +194,44 @@ development would include:
 -  Possibly moving some infrequently used backends to third-party
    packages.
 
+Flexible higher-level data structures
```

Comment:
Shouldn't there also be a bit about `xarray.Variable`?

Reply:
added in 0a04faf